actors/builtin/miner/miner_actor.go

```diff
@@ -528,7 +545,7 @@ func (a Actor) PreCommitSector(rt Runtime, params *SectorPreCommitInfo) *adt.Emp
 
 type ProveCommitSectorParams struct {
 	SectorNumber abi.SectorNumber
-	Proof        []byte
+	Proof        []byte // PARCHECK: arbitrary size
```

Comment:
It's usually fine to allow arbitrary size proofs. However, I'm a bit concerned in this case because prove-commit proofs are batched for async processing. That means we're not going to validate until later.

Reply:
Done.